tests/benchmarks/comparison.sh

```diff
@@ -1,4 +1,6 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
+cd $(dirname $0)
```

Comment:
This will break if the path contains spaces.

It's also better form to be using "${BASH_SOURCE[0]}", which contains the proper path when sourcing the file with `. ` or `source`. Shouldn't matter too much in this case, but it's good to keep in mind.

I would recommend using the following instead:

```bash
cd "$(dirname "${BASH_SOURCE[0]}")"
```

Reply:
I didn't know that, thanks :)